# HG changeset patch
# User Thomas Weber <tweber@debian.org>
# Date 1298229183 0
#      Sun Feb 20 19:13:03 2011 +0000
# Node ID 32279948bf3bbd537bb00b3f7c3ce7a4145faf52
# Parent  f5a5e5252b5ae9f946349aba2568a216113e7f64
Document the Octave:autoload-relative-file-name warning.

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -50,16 +50,21 @@ 2010-02-22  Rik  <octave@nomad.inbox5.co
 2010-02-20  Karsten Trulsen  <karstent@math.uio.no>
 
 	* signal/fftshift.m: Fix broken function.  Bug 32442.
 
 2010-02-21  Carlo de Falco  <kingcrimson@tiscali.it>
 
 	* pkg/get_forge_pkg.m: Fix typo in new PCRE style regular expression.
 
+2011-02-20  Thomas Weber <tweber@debian.org>
+
+	* miscellaneous/warning_ids.m: Document
+	Octave::autoload-relative-file-name warning.
+
 2010-02-19  Rik  <octave@nomad.inbox5.com>
 
 	* io/dlmwrite.m, pkg/get_forge_pkg.m, plot/__gnuplot_ginput__.m,
 	plot/__go_draw_axes__.m, testfun/runtests.m: Use PCRE regular
 	expressions to simplify scripts.
 
 2011-02-19  Ben Abbott  <bpabbott@mac.com>
 
diff --git a/scripts/miscellaneous/warning_ids.m b/scripts/miscellaneous/warning_ids.m
--- a/scripts/miscellaneous/warning_ids.m
+++ b/scripts/miscellaneous/warning_ids.m
@@ -104,16 +104,25 @@
 ## @item Octave:associativity-change
 ## If the @code{Octave:associativity-change} warning is
 ## enabled, Octave will warn about possible changes in the meaning of
 ## some code due to changes in associativity for some operators.
 ## Associativity changes have typically been made for @sc{matlab}
 ## compatibility.  By default, the @code{Octave:associativity-change}
 ## warning is enabled.
 ##
+## @item Octave:autoload-relative-file-name
+## If the @code{Octave:autoload-relative-file-name} is enabled,
+## Octave will warn when parsing autoload() function calls with relative
+## paths to function files. This usually happens when using autoload()
+## calls in PKG_ADD files, when the PKG_ADD file is not in the same
+## directory as the .oct file referred to by the autoload() command.
+## By default, the @code{Octave:autoload-relative-file-name}
+## warning is enabled.
+##
 ## @item Octave:divide-by-zero
 ## If the @code{Octave:divide-by-zero} warning is enabled, a
 ## warning is issued when Octave encounters a division by zero.  By
 ## default, the @code{Octave:divide-by-zero} warning is enabled.
 ##
 ## @item Octave:empty-list-elements
 ## If the @code{Octave:empty-list-elements} warning is enabled, a
 ## warning is issued when an empty matrix is found in a matrix list.
